.github/workflows/ci.yml

```
@@ -24,33 +27,26 @@ jobs:
       fail-fast: false
       matrix:
         # Nextflow versions
-        include:
-          # Test pipeline minimum Nextflow version
-          - NXF_VER: "21.10.3"
-            NXF_EDGE: ""
-          # Test latest edge release of Nextflow
-          - NXF_VER: ""
-            NXF_EDGE: "1"
+        NXF_VER:
+          - "21.10.3"
+          - "latest-everything"
         test_profile: ["test_lfq", "test_dia", "test_localize", "test_tmt"]
         exec_profile: ["docker", "conda"]
         exclude:
           - test_profile: test_dia
             exec_profile: conda
           - test_profile: test_localize
             exec_profile: conda
+          - NXF_VER: "latest-everything"
+            profile: "conda"
```

Maybe it should it be exec_profile instead?

```suggestion
            exec_profile: "conda"
```

---

[Agreed](https://github.com/nf-core/quantms/pull/53#issuecomment-1244103014) 😅